Post/tests.py

```diff
@@ -1,3 +1,12 @@
-# from django.test import TestCase
+from Post.models import post
 
-# Create your tests here.
+
+class TestPost():
+
+    def test_post_creation(self):
```

Comment:
Why no using @pytest.mark.django_db? 
similar to here? https://redhat-beyond.slack.com/archives/C01D9TBDTKM/p1618847739037000?thread_ts=1618738675.026800&cid=C01D9TBDTKM
Page 30 - 33

Reply:
Because you do not need a db for this specific check.
If you exercise a scenario where a db is needed, you must specify it using the mark, otherwise the test will fail.

I hope to see such a test being added in a following PR.

Reply:
Hi Yariv and Ed,

I have two questions-
- Do I need to make changes to the tests file in this PR and to open a new PR for tests that use the DB? or should I change it in this PR?
- Do I need to change the pipfile to install only the django-pytest?

Thank you!

Reply:
>     * Do I need to make changes to the tests file in this PR and to open a new PR for tests that use the DB? or should I change it in this PR?

I am good with this simple tests, mainly because this PR is about adding the infrastructure and not testing the app.
But maybe @Yarboa will have other thoughts about this.

The only small thing that bothers me here is the commit separation. In the first commit you add a test that is not that good and then you change it in the next commit. You could squash the two or just write the test in the second all together (leaving the first without it). You could also have in the first commit just a placeholder with an `assert True` or something similar. Up to you.

>     * Do I need to change the pipfile to install only the django-pytest?

I consider this a matter of taste, you can choose one of the approaches and defend it.
The end result for both is the same.

Reply:
Hi @EdDev,
I changed the commit as you asked.